src/js/camresolutionstest.js

```diff
@@ -175,16 +175,16 @@ CamResolutionsTest.prototype = {
     var statsReport = {};
     var frameStats = frameChecker.frameStats;
 
-    for (var index = 0; index < stats.length - 1; index++) {
+    for (var index in stats) {
```

Comment:
Probably using the same that adapter.js uses is good enough. As if there is a bug there then we would have the same.